demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -979,34 +989,111 @@ export class DemoMeetingApp
+    contentRedactionCb.addEventListener('click', () => {
+      (document.getElementById('content-identification-checkbox') as HTMLInputElement).disabled = contentRedactionCb.checked;
+      (document.getElementById('transcribe-entity-types') as HTMLInputElement).classList.toggle('hidden', !contentRedactionCb.checked);
+    });
+
+    const partialResultsStabilityCb = document.getElementById('partial-stabilization-checkbox') as HTMLInputElement;
+    partialResultsStabilityCb.addEventListener('click', () => {
+      (document.getElementById('transcribe-partial-stability').classList.toggle('hidden', !partialResultsStabilityCb.checked));
+    });
+
+    const languageModelCb = document.getElementById('custom-language-model-checkbox') as HTMLInputElement;
+    languageModelCb.addEventListener('click', () => {
+      (document.getElementById('language-model').classList.toggle('hidden', !languageModelCb.checked));
+    });
+
     const buttonStartTranscription = document.getElementById('button-start-transcription');
     buttonStartTranscription.addEventListener('click', async () => {
       let engine = '';
       let languageCode = '';
       let region = '';
+      const transcriptionStreamParams:TranscriptionStreamParams = {};
       if ((document.getElementById('engine-transcribe') as HTMLInputElement).checked) {
         engine = 'transcribe';
         languageCode = (document.getElementById('transcribe-language') as HTMLInputElement).value;
         region = (document.getElementById('transcribe-region') as HTMLInputElement).value;
+        let contentIdentification = (document.getElementById('content-identification-checkbox') as HTMLInputElement).checked;
+        if (contentIdentification) {
+          transcriptionStreamParams.contentIdentificationType = 'PII';
+        }
+        let contentRedaction = (document.getElementById('content-redaction-checkbox') as HTMLInputElement).checked;
+        if (contentRedaction) {
+          transcriptionStreamParams.contentRedactionType = 'PII';
+        }
+        let partialStabilization = (document.getElementById('partial-stabilization-checkbox') as HTMLInputElement).checked;
+        if (partialStabilization) {
+          transcriptionStreamParams.enablePartialResultsStability = partialStabilization;
+        }
+        let partialStabilityFactor = (document.getElementById('partial-stability') as HTMLInputElement).value;
+        if (partialStabilityFactor) {
+          transcriptionStreamParams.partialStabilityFactor = partialStabilityFactor;
+        }
+        if (contentIdentification || contentRedaction) {
+          const selected = document.querySelectorAll('#transcribe-entity option:checked');
+          let values = '';
+          if (selected.length > 0) {
+            values = Array.from(selected).map(el => (el as HTMLInputElement).value).join(',');
+            if (values.startsWith(',')) {
+              values = values.substr(1);
```

Comment:
This seems like a tricky way to phrase what you're doing here. Why are you not checking the first element before you `join`?